src/main/java/backend/github/ApiQuotaInfo.java

```diff
@@ -0,0 +1,32 @@
+package backend.github;
+
+import util.Utility;
+
+/**
+ * Holds GitHub API quota information. Data are immutable.
+ */
+public class ApiQuotaInfo {
+
+    /**
+     * The remaining API quota in the current window.
+     */
+    private final int remainingQuota;
+
+    /**
+     * The time at which the current API quota window resets in UTC epoch milliseconds.
+     */
+    private final long nextRefreshInMillisecs;
```

Comment:
We use the term `refresh` for another meaning as well (i.e. to mean refreshing of issue data). To avoid confusion, this should be named `millisecsToNextTopup`?